js/script.js

```diff
@@ -1,39 +1,43 @@
 // Gets a reference to the form element, assuming
 // it contains the ID attribute "contact-form".
-var form = document.getElementById('contact-form');
+const form = document.getElementById('contact-form');
+const email = form.querySelector('input[type="email"]');
 
 // Adds a listener for the "submit" event.
-form.addEventListener('submit', function(event) {
-
+form.addEventListener('submit', (event) => {
   // Prevents the browser from submitting the form
   // and thus unloading the current page.
-  event.preventDefault();
-
-  // Creates a timeout to call submitForm after one second.
-  setTimeout(submitForm, 1000);
-
-  // Monitors whether or not the form has been submitted.
-  // This prevents the form from being submitted twice in cases
-  // where the event callback function fires normally.
-  var formSubmitted = false;
-
-  function submitForm() {
-    if (!formSubmitted) {
-      formSubmitted = true;
-      form.submit();
+  console.log('event listener');
+  if (!checkLower(email.value)) {
+    event.preventDefault();
+  } else {
+    // Creates a timeout to call submitForm after one second.
+    // eslint-disable-next-line no-use-before-define
+    setTimeout(submitForm, 1000);
+
+    // Monitors whether or not the form has been submitted.
+    // This prevents the form from being submitted twice in cases
+    // where the event callback function fires normally.
+    let formSubmitted = false;
+
+    function submitForm() {
+      if (!formSubmitted) {
+        formSubmitted = true;
+        form.submit();
+      }
     }
-  }
 
-  // Sends the event to Google Analytics and
-  // resubmits the form once the hit is done.
-  gtag('event', 'signup_form_complete', {
-    'event_callback': submitForm
-  });
+    // Sends the event to Google Analytics and
+    // resubmits the form once the hit is done.
+    gtag('event', 'signup_form_complete', {
+      event_callback: submitForm,
+    });
+  }
 });
 
-
+// eslint-disable-next-line camelcase
 function get_resume() {
```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them. you can fix this by simply changing the function name from `get_resume` to `getResume`